apps/okreads-e2e/src/integration/search-books.spec.ts

```diff
@@ -4,14 +4,35 @@ describe('When: Use the search feature', () => {
   });
 
   it('Then: I should be able to search books by title', () => {
-    cy.get('input[type="search"]').type('javascript');
+    cy.get('input[type="search"]').type('angular');
 
     cy.get('form').submit();
 
     cy.get('[data-testing="book-item"]').should('have.length.greaterThan', 1);
   });
 
-  xit('Then: I should see search results as I am typing', () => {
-    // TODO: Implement this test!
+  it('Then: I should be able to undo add book', () => {
+    cy.get('input[type="search"]').type('angular');
+
+    cy.get('form').submit();
+
+    let isClicked = false;
+    cy.get('[data-testing="cta-add-to-reading-list"]').each((ele, index, list) => {
+      if (ele[0].innerText === 'Want to Read' && !isClicked) {
+        ele.click();
+        isClicked = true;
+      }
+    });
```

Comment:
Same as above, you should be able to get the first item without going through for loop